Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile is used only for API tests.
 
-FROM golang:1.20
+FROM golang:1.21
```

Comment:
I believe we should update the version in all go.mod files at the same time.

Reply:
Not really, it can be done in different PR and later.

Reply:
Just checked, `brew` still doesn't have `1.21` in its repositories, so we should definitely wait some time before doing that.

Reply:
+1